core/storage/user/gae_models.py

```diff
@@ -1806,6 +1906,12 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE
 
+    @staticmethod
+    def get_export_policy():
+        """Model contains user data."""
+        return base_models.EXPORT_POLICY.NOT_APPLICABLE
```

Comment:
This return value is wrong. A takeout function needs to be written for it and this should be added to the takeout doc. (The model was added in late 2019.)

In general please actually look at the fields of the models before setting NOT_APPLICABLE; don't do this in a cookie-cutter fashion. Some models may have been written after the initial batch of takeout functionality was implemented, and we need to catch those.

Reply:
Updated to have TO_BE_IMPLEMENTED.